### PR TITLE
Update View, Edit, and Exit commands in UserGuide.md 

### PR DESCRIPTION
**Pull request by brandon-nam**
Add view command, edit edit and exit commands, fix bolding issue

### CODE REVIEW

File: docs/UserGuide.md
Comment (butteredyakiimo) [inline docs/UserGuide.md]: @nananakx-x may have made some changes to the format!  I think stuff like availability and housing type may not be tags anymore

File: docs/UserGuide.md
Comment (butteredyakiimo) [inline docs/UserGuide.md]: maybe a clearer way to phrase this line can be: "...but the number of parameters to be edited can vary from zero to all fields." what do you think!

File: docs/UserGuide.md
Comment (butteredyakiimo) [inline docs/UserGuide.md]: I think can add a "the" before "normal foster family list view" and a  comma after "while on the profile view page"

**Review by butteredyakiimo — COMMENTED** [review]
I think you will need to check with Nanette on the tagging part being changed to parameters! I assume you haven't added in the edit part in view because its not implemented yet, so I will ignore that first haha. Other than that, LGTM

File: docs/UserGuide.md
Comment (nananakx-x) [reply docs/UserGuide.md]: Yup, they are no longer tags! I will update the add command UG part after we merge everything, then I'll inform brandon about the changes if theres a need for clarification. Otherwise, the last message on tele about the add command is correct.

File: docs/UserGuide.md
Comment (nananakx-x) [inline docs/UserGuide.md]: I think we can discuss this again; can either delete all data as said above, or maybe we show an error message or something...?

**Review by nananakx-x — DISMISSED** [review]
I think this can be merged after we merge the rest, the other information for add command (which will affect edit) I will add in later and we can change the current description accordingly!

**Pull request closed without merging**
